### DISCUSSION TITLE
graphql-ws to graphql-transport-ws Protocol work

### DISCUSSION THREAD

**Discussion by djpritchett** [datadvance/DjangoChannelsGraphqlWs#114] [General]
big thumbs up for @prokher for updating ([switch-from-graphql-transport-ws-to-graphql-ws-subprotocol])(https://github.com/datadvance/DjangoChannelsGraphqlWs/tree/switch-from-graphql-transport-ws-to-graphql-ws-subprotocol)) the most confusing issue I have ran into in a long while graphql-transport-ws TO graphql-ws subprotocol

which as we know means graphql-ws  to graphql-transport-ws

seriously though, thank you

**Comment by prokher** [comment]
Thank you for warm words. I happy to redirect these words to my team: @sergey-komissarov, @AlexzAK, @denis.sheronov, @herclogon. I believe we will integrate these changes into the master branch and issue a new RC quite soon.
